cpp20_synchronization_library.bs

```diff
@@ -358,7 +355,7 @@ void wait(T old, memory_order order = memory_order::seq_cst) const noexcept;
 <div class="numbered">
 *Effects:* Repeatedly performs the following steps, in order: 
 
-  - Evaluates `object->load(order) != old` then, if the result is `true`, returns.
+  - Evaluates `this->load(order) != old` then, if the result is `true`, returns.
```

Comment:
We should probably remove `this->` everywhere in the member functions, e.g. `load(order) != old)`